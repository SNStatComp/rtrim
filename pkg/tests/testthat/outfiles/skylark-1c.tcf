FILE outfiles/skylark.dat
TITLE  skylark-1c
NTIMES 8
NCOVARS 2
LABELS
Habitat
Cov2
End
MISSING 999
WEIGHT Absent

comment Example 1; both overdispersion and serialcorrelation
weighting off
overdisp on
serialcor on
model 3
