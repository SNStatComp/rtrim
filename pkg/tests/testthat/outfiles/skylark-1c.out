 TRIM 3.61 :  TRend analysis and Indices for Monitoring data 
 STATISTICS NETHERLANDS 

 Date/Time: 4-7-2016 15:07:34

 Title :   skylark-1c

 Comment: Example 1; both overdispersion and serialcorrelation

 The following  5 variables have been read from file: 
 F:\TRIM\TRIM_manual_demo\skylark.dat

 1. Site             number of values:    55
 2. Time             number of values:     8
 3. Count            missing =        999
 4. HABITAT          number of values:  2
 5. COV2             number of values:  4

 Number of sites without positive counts (removed)       0

 Number of observed zero counts           0
 Number of observed positive counts     202
 Total number of observed counts        202
 Number of missing counts               238
 Total number of counts                 440

 Total count                         2536.0

 Sites containing more than 10% of the total count 
   Site Number  Observed Total    %   
         3        431.0         17.0
        37        266.0         10.5
        40        624.0         24.6

 Time Point Averages 
   TimePoint  Observations  Average   Index  
       1              25       8.52    1.00
       2              20       8.10    0.95
       3              30      10.90    1.28
       4              30      11.27    1.32
       5              28      12.75    1.50
       6              29      14.66    1.72
       7              22      17.00    2.00
       8              18      18.89    2.22

 RESULTS FOR MODEL: Effects for Each Time Point
 ----------------------------------------------


 ESTIMATION METHOD = Generalised Estimating Equations


  Total time used:  2.84 seconds

  Estimated Overdispersion     =  1.367
  Estimated Serial Correlation =  0.302

 GOODNESS OF FIT
  Chi-square                191.40, df    140, p 0.0026
  Likelihood Ratio          194.80, df    140, p 0.0015
  AIC (up to a constant)    -85.20

 WALD-TEST FOR SIGNIFICANCE OF DEVIATIONS FROM LINEAR TREND 
  Wald-Test             16.22, df     6, p  0.0126

 PARAMETER ESTIMATES 

 Parameters for Each Time Point

  Time    Additive    std.err.   Multiplicative  std.err.
     1      0                          1
     2     -0.3202      0.1055         0.7260      0.0766
     3     -0.1687      0.1054         0.8448      0.0891
     4     -0.1897      0.1083         0.8272      0.0896
     5     -0.0824      0.1070         0.9209      0.0986
     6      0.0208      0.1059         1.0210      0.1081
     7      0.0997      0.1082         1.1048      0.1196
     8      0.1558      0.1108         1.1686      0.1295

 Linear Trend + Deviations for Each Time 

           Additive    std.err.   Multiplicative  std.err.
 Slope      0.0460      0.0142         1.0471      0.0149

 Time Deviations
    1       0.2217      0.0582         1.2481      0.0727
    2      -0.1446      0.0660         0.8654      0.0571
    3      -0.0391      0.0575         0.9617      0.0553
    4      -0.1061      0.0588         0.8994      0.0529
    5      -0.0448      0.0546         0.9561      0.0522
    6       0.0124      0.0488         1.0124      0.0495
    7       0.0452      0.0452         1.0463      0.0473
    8       0.0553      0.0495         1.0569      0.0523

 Time INDICES
  Time     Model     std.err.     Imputed    std.err.
     1     1                       1
     2     0.7260     0.0766       0.7201     0.0766
     3     0.8448     0.0891       0.8454     0.0893
     4     0.8272     0.0896       0.8314     0.0901
     5     0.9209     0.0986       0.9221     0.0989
     6     1.0210     0.1081       1.0251     0.1086
     7     1.1048     0.1196       1.1082     0.1201
     8     1.1686     0.1295       1.1828     0.1308

 TIME TOTALS
  Time      Model    std.err.     Imputed    std.err.
     1        509         45          509         45
     2        370         35          366         35
     3        430         29          430         29
     4        421         28          423         28
     5        469         31          469         31
     6        520         32          521         32
     7        563         37          564         37
     8        595         42          601         42

 OVERALL SLOPE MODEL: Moderate increase (p<0.01) **
  Additive      std.err.   Multiplicative  std.err.
    0.0460       0.0142         1.0471      0.0149

 OVERALL SLOPE IMPUTED+(recommended): Moderate increase (p<0.01) **
  Additive      std.err.   Multiplicative  std.err.
    0.0478       0.0142         1.0489      0.0149

